fe/fe-core/src/main/java/com/starrocks/catalog/ScalarType.java

```diff
@@ -487,12 +486,11 @@ public static ScalarType getAssignmentCompatibleType(
      * Returns true t1 can be implicitly cast to t2, false otherwise.
      * If strict is true, only consider casts that result in no loss of precision.
      */
-    public static boolean isImplicitlyCastable(
-            ScalarType t1, ScalarType t2, boolean strict) {
+    public static boolean isImplicitlyCastable(ScalarType t1, ScalarType t2, boolean strict) {
         return getAssignmentCompatibleType(t1, t2, strict).matchesType(t2);
     }
 
-    public static boolean canCastTo(ScalarType type, ScalarType targetType) {
+    public static boolean isExplicitCastable(ScalarType type, ScalarType targetType) {
         return PrimitiveType.isImplicitCast(type.getPrimitiveType(), targetType.getPrimitiveType());
```

Comment:
Why `isExplicitCastable` call `isExplicitCastable`?  which seems strange

Reply:
Emm... Definitely strange. 
The caller of this method needs to check explicit cast, whether the implementation is actually implicit cast.

I have not figure out the reason and solution, so I could reserve the name, and leave a TODO here.